__tests__/features/end-to-end.steps.ts

```diff
@@ -505,6 +510,30 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
         })
       ).sendKeys(processData.residents[presentTenantRef].carer.notes);
 
+      await browser!.submit();
+      //other support page
```

Comment:
```suggestion

      // Other support page
```

Definitely a nitpick, but if you're here...